live-examples/js-examples/string/string-tolocaleuppercase.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var cedilla = '\u00E7';
+
+console.log(cedilla.toLocaleUpperCase('en-US'));
+// expected output: "Ç"
+
+console.log(cedilla.toLocaleUpperCase('TR'));
+// expected output: "Ç"
+</code>
```

Comment:
This seems to give the same output for both locales to me (and the same output as toUpperCase as well) so it doesn't seem to be a good demonstration of the API. We could do the same as toLocaleLowerCase, but in reverse:

```
var city = 'istanbul';

console.log(city.toLocaleUpperCase('en-US'));
// expected output: "ISTANBUL"

console.log(city.toLocaleUpperCase('TR'));
// expected output: "İSTANBUL"
```
What do you think? I might be missing some subtlety here though...

Reply:
Changed to your example in my other pull request. Much better. Thank you.